spec/rspec/matchers/composable_spec.rb

```diff
@@ -12,6 +12,21 @@ module Matchers
         }.to fail_with(STDOUT.inspect)
       end
 
+      it "does not blow up when surfacing descriptions from an unreadable Range object" do
+        infinity ||= (1.0/0.0)
```

Comment:
Sorry another thing I didn't mean to commit, I was originally attempting to set the constant if it didn't exist (1.8.7 basically)